files/es/games/techniques/2d_collision_detection/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Detección de colisiones 2D
+slug: Games/Techniques/2D_collision_detection
+translation_of: Games/Techniques/2D_collision_detection
+original_slug: Games/Techniques/2D_collision_detection
+---
+{{GamesSidebar}}
+
+Los algoritmos para detectar colisiones en juegos 2D dependen del tipo de formas que pueden colisionar (p. ej., Rectángulo con rectángulo, Rectángulo con círculo, Círculo con círculo). En general, tendrá una forma genérica simple que cubre la entidad conocida como _"hitbox"_, por lo que, aunque la colisión no sea perfecta en píxeles, se verá lo suficientemente bien y tendrá un rendimiento en varias entidades. Este artículo proporciona una revisión de las técnicas más comunes utilizadas para proporcionar detección de colisiones en juegos 2D.
```

Comment:
```suggestion
Los algoritmos para detectar colisiones en juegos 2D dependen del tipo de formas que pueden colisionar (p. ej., Rectángulo con rectángulo, Rectángulo con círculo, Círculo con círculo). En general, tendrá una forma genérica simple que cubre la entidad conocida como _"hitbox"_, por lo que, aunque la colisión no sea perfecta en píxeles, se verá lo suficientemente bien y tendrá un rendimiento eficiente en varias entidades. Este artículo proporciona una revisión de las técnicas más comunes utilizadas para proporcionar detección de colisiones en juegos 2D.
```
_"be performant"_ Curiosamente, la palabra _performant_ en inglés [no existe](https://english.stackexchange.com/questions/38945/what-is-wrong-with-the-word-performant).  Fue hasta hace no mucho que se introdujo únicamente en el diccionario de [Cambridge](https://dictionary.cambridge.org/us/dictionary/english/performant)

Reply:
Aplicado, muchas gracias!